.gitignore

```diff
@@ -8,3 +8,5 @@
 terraform-provider-honeycombio*
 
 coverage.txt
+
+.idea
```

Comment:
Updating the .gitignore to handle every person's personal setup will lead to a lot of entries here, can I propose using a global gitignore to handle your personal preferences?

Relevant resource: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c